pkg/vdri/trustbloc/vdri.go

```diff
@@ -121,18 +126,16 @@ func (v *VDRI) Read(did string, opts ...vdriapi.ResolveOpts) (*docdid.Doc, error
 	var doc *docdid.Doc
 
 	for _, e := range endpoints {
-		sideTreeVDRI, err := v.getHTTPVDRI(e.URL)
+		resp, err := v.sidetreeResolve(e.URL, did, opts...)
```

Comment:
All sidetreeResolve's should be done in parallel, i.e. separate Go routine for performance reasons. (Probably should be a separate PR.)